metaflow/metaflow_config.py

```diff
@@ -201,6 +201,8 @@ def _validate_choice(name, value):
 )
 CARD_NO_WARNING = from_conf("METAFLOW_CARD_NO_WARNING", False)
 
+SKIP_CARD_DUAL_WRITE = from_conf("METAFLOW_SKIP_CARD_DUALWRITE", False)
```

Comment:
This works for me but as a quick note, this setting would drop off at some point (so maybe we should put a note that it will become True by default and then drop off).